esti/main_test.go

```diff
@@ -70,7 +70,7 @@ func TestMain(m *testing.M) {
 		StorageNS: "esti-system-testing",
 	}
 	if *useLocalCredentials {
-		params.AdminAccessKeyID = "AKIAIOSFODNN7EXAMPLE"
+		params.AdminAccessKeyID = "AKIAIOSFDNN7EXAMPLEQ"
```

Comment:
Why are you changing our defaults?

Reply:
It fails the doctor tests as it is not according to spec

Reply:
Nice one :)